docs/ru/query_language/functions/date_time_functions.md

```diff
@@ -129,6 +227,71 @@ SELECT
 Округляет время до получаса.
 Эта функция является специфичной для Яндекс.Метрики, так как пол часа - минимальное время, для которого, если соседние по времени хиты одного посетителя на одном счётчике отстоят друг от друга строго более, чем на это время, визит может быть разбит на два визита. То есть, кортежи (номер счётчика, идентификатор посетителя, тайм-слот) могут использоваться для поиска хитов, входящий в соответствующий визит.
 
+## toYYYYMM
+Переводит дату или дату со временем в число типа UInt32, содержащее номер года и месяца (YYYY * 100 + MM).
+
+## toYYYYMMDD
+Переводит дату или дату со временем в число типа UInt32, содержащее номер года, месяца и дня (YYYY * 10000 + MM * 100 + DD).
+
+## toYYYYMMDDhhmmss
+Переводит дату или дату со временем в число типа UInt64 содержащее номер года, месяца, дня и время (YYYY * 10000000000 + MM * 100000000 + DD * 1000000 + hh * 10000 + mm * 100 + ss).
+
+## addYears, addMonths, addWeeks, addDays, addHours, addMinutes, addSeconds, addQuarters
+Добавляет интервал Date/DateTime к Date/DateTime, а затем возвращает Date/DateTime. 
```

Comment:
But how do these functions work?
What will happen if we will add one month to 31th January?